### PR TITLE
Add astral.cool presence

### PR DESCRIPTION
**Pull request by aspectiscool**
image
image

### CODE REVIEW

File: websites/A/Astral/presence.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    case path.endsWith("/account"):
+        presenceData.details = "Viewing their account";
+        presenceData.startTimestamp = Date.now();
+        break;
+    case !isNaN(parseInt(path.charAt(path.length - 1))):
+        presenceData.details = "";
+        presenceData.state = `Viewing ${document.getElementsByClassName("title___aqmzM")[0].innerHTML}"s profile`;
+        presenceData.startTimestamp = Date.now();
+        break;
+  }
```
Comment (ririxidev) [inline websites/A/Astral/presence.ts:70]: don't use switch(true) pls, if path will change it will detect it...

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/A/Astral/presence.ts [hunk 2]
```diff
@@ -12,61 +12,50 @@ presence.on("UpdateData", async () => {
...
+    presenceData.startTimestamp = Date.now();
+  } else if(path.endsWith("/shorten")) {
+    presenceData.state = "Shortening a link";
+    presenceData.details = "Viewing the shortener";
+    presenceData.startTimestamp = Date.now();
+  } else if (path.endsWith("/account")) {
+    presenceData.details = "Viewing their account";
+    presenceData.startTimestamp = Date.now();
+  } else if (!isNaN(parseInt(path.charAt(path.length - 1)))) {
+    presenceData.details = "";
```
Comment (Bas950) [inline websites/A/Astral/presence.ts:56]: dont set empty details

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/Astral/presence.ts [hunk 3]
```diff
@@ -23,15 +23,12 @@ presence.on("UpdateData", async () => {
     domainPicker = document.getElementsByClassName("ant-select-open")[0];
 
    if (embedEditor && embedEditor.parentElement.style.display !== "none") {
-       presenceData.details = "";
        presenceData.state = "Editing their embed";
```
Comment (Bas950) [inline websites/A/Astral/presence.ts]: instead of only setting state, only set the details.
*some on all those other edits*

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
no response from pull requeqst creator. closing 🤷

**Pull request closed without merging**